peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+and the user interface is not very good (and not going to be improved, according
+to Microsoft). Without managing these, it is relatively easy for an undesired
+version to be launched, though in general the targets can only be changed
+manually by the user, and not by merely installing another app.
+
+Both the Nuget package and the embeddable distro are as simple and reliable to
+install as extracting an archive file, though it's worth noting that for many
+Python users this is not a common task. They provide no install management at
+all, and cannot be reliably updated other than by deleting and re-extracting.
+User expectations that are un-met are almost always due to users selecting the
+wrong installer. Both these packages are for specialised cases, and while they
+are documented as such, the attraction of a plain ZIP file leads some users into
+failure.
+
+Overview of PyManager
+=====================
+
+("PyManager" name open for bikeshedding)
+
+PyManager is the internal name of our proposed replacement installer tool. It
+will be distributed both in the Windows Store and on python.org as an MSIX
+package. Downloading from either source will get an identical package, and
+both will support automatic updates (through the Store) for new releases.
+
+The user visible name will be "Python Install Manager", published by the
+Python Software Foundation. After publishing, we will request that Microsoft
+adjust their ``python.exe`` stub to open to this new app.
+
+This app does not directly provide a version of Python, but it does provide the
+global commands that users expect to work, as well as file associations and
+Start menu shortcuts. The OS will prompt users to launch the app after install,
+which will trigger an automatic install of the current release of CPython and
+then launch it. From the user's perspective, they have the same initial
+experience as today, with one added progress bar on first launch.
+
+The global commands provided by the app must be static and bundled into the app
+itself. They can only change their behaviour at runtime, and cannot be
+redirected to different executables except by the user (and then only to another
+installed app). So the commands to be provided by PyManager are ``python.exe``,
+``python3.exe``, ``py.exe``, ``pymanager.exe``. Each of these must have the
+ability to inspect the user's system and choose the correct runtime to launch.
+Additionally, ``py`` and ``pymanager`` will have management subcommands to allow
+adding and removing runtimes.
+
+In line with PEP 394 and the default behaviour of Windows, the recommended
+command for launching Python is ``python.exe``. As provided by PyManager, this
+will locate an existing install, either among those that PyManager manages or
+using PEP 514, or it will install the latest available version of CPython and
+select that. The ``python3.exe`` command behaves similarly, but is only allowed
+to find 3.x installs from python.org.
```

Comment:
```suggestion
In line with :pep:`394` and the default behaviour of Windows, the recommended
command for launching Python is ``python.exe``. As provided by PyManager, this
will locate an existing install, either among those that PyManager manages or
using :pep:`514`, or it will install the latest available version of CPython and
select that. The ``python3.exe`` command behaves similarly, but is only allowed
to find 3.x installs from python.org.
```